buildSrc/build.gradle

```diff
@@ -133,6 +133,10 @@ dependencies {
   integTestImplementation('org.spockframework:spock-core:2.3-groovy-3.0') {
     exclude module: "groovy"
   }
+  implementation('org.ajoberstar.grgit:grgit-core:5.2.0') {
```

Comment:
I am wondering if we need this dependency at all, it seems like we don't use JGit anywhere

Reply:
Thanks @reta! Good catch, just checked there's no usage so closing this out. :)

Reply:
We don't use JGit directly, but we do use grgit-core right? Looks like it was added [here](https://github.com/opensearch-project/OpenSearch/blob/23fd9a5d94a17459c8953ccd3e089770dc1d1700/buildSrc/src/main/groovy/org/opensearch/gradle/CheckCompatibilityTask.groovy#L12).

Reply:
Thanks @andrross!